atlas/src/shared/components/VideoPreview/VideoPreview.tsx

```diff
@@ -71,6 +72,25 @@ const VideoPreview: React.FC<VideoPreviewProps> = ({
     onClick(e)
   }
 
+  const MIN_COVER_WIDTH = 300
+  const MAX_COVER_WIDTH = 600
+  const MIN_SCALING_FACTOR = 1
+  const MAX_SCALING_FACTOR = 1.375
+  // Linear Interpolation, see https://en.wikipedia.org/wiki/Linear_interpolation
+  const calculateScalingFactor = (coverWidth: number) =>
+    MIN_SCALING_FACTOR +
+    ((coverWidth - MIN_COVER_WIDTH) * (MAX_SCALING_FACTOR - MIN_SCALING_FACTOR)) / (MAX_COVER_WIDTH - MIN_COVER_WIDTH)
+
+  const [scalingFactor, setScalingFactor] = useState(MIN_SCALING_FACTOR)
+  const { ref: imgRef } = useResizeObserver<HTMLImageElement>({
```

Comment:
Since we're using hooks here I think it'd be best to move this code fragment above the click handlers. In general, I think we should follow the following component code layout:
1. Hooks
2. Event handlers
3. Return

Reply:
Good Idea. I've moved the hooks to the start of the component and the interpolation logic out of it since its static.

Reply:
Looks much better now :+1: